updates.rst

```diff
@@ -4,20 +4,22 @@
 Updates
 =======
 
-.. highlight:: bash
+NethSecurity allows two types of updates, both available from the ``Update`` section under the ``System`` menu.
 
-NethSecurity uses two different methods to update the system:
+**1. Bug & security fixes**
 
-**1. Security and minor bugfixes are distributed using packages.** To update the system, run the following commands: ::
+These updates are intended for minor updates and bugfixes.
 
-    opkg update && opkg upgrade
+Typically they are performed automatically, but at any time it is possible to check for new updates available by clicking on :guilabel:`Check for Fixes` button.
+These updates do not require a restart of NethSecurity, they are tied to a specific version and distributed using packages.
 
-The ``opkg update`` command updates the list of available packages, and the ``opkg upgrade`` command upgrades all of the installed packages to the latest versions.
+**2. System updates** (switching to a different version)
 
-**2. New features are released with new images.** To update the system to a new version with new features, download the new image file and run the following command: ::
+These updates involve the transition to a new version of the firmware that introduces new features, improvements and a greater hardware support.
 
-  sysupgrade -v nethsecurity-<version>-x86-64-generic-squashfs-combined.img.gz
```

Comment:
Are we sure we want to remove this information? It may be useful.

Reply:
might it be better to put it in the dev manual instead of this manual?
https://dev.nethsecurity.org

Reply:
It's useful to users, not to developers.

Reply:
I have nothing against it